var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -33,6 +33,7 @@ class Dyninst(Package):
     url = "https://github.com/dyninst/dyninst/archive/v9.2.0.tar.gz"
     list_url = "http://www.dyninst.org/downloads/dyninst-8.x"
 
+    version('develop', git="https://github.com/dyninst/dyninst.git", branch='master')
```

Comment:
This makes the master branch the default version if a user doesn't specify otherwise. Perhaps we should pin the preferred to the latest numbered version?

Reply:
Doesn't Spack have special treatment for `develop` such that it is newer than all other versions but still isn't chosen by default? I thought we default to the newest numeric release?

Reply:
sorry, yes it looks like it is properly grabbing the latest numbered version by default.